decidim-core/app/packs/src/decidim/index.js

```diff
@@ -216,6 +216,18 @@ $(() => initializer());
 document.addEventListener("remote-modal:loaded", ({ detail }) => initializer(detail));
 document.addEventListener("ajax:loaded", ({ detail }) => initializer(detail));
 
+// document.addEventListener("ajax:success", () => {
```

Comment:
Why this is commented? Is to be used in another PR? I prefer to just remove it and add it in the next PR